static/sass/header.scss

```diff
@@ -0,0 +1,28 @@
+// based on p-navigation--light pattern from Vanilla
+@mixin navigation-x-light {
+  .p-navigation--x-light {
```

Comment:
Vanilla doesn't have pattern for white background heading, so adding it here.
Based on built in patterns not to write too much custom styles.